### ISSUE TITLE
expanded command docs

### ISSUE THREAD

**Issue by jordan2lee** [calypr/calypr.github.io#20]
generate pages for each g3t command with in depth descriptions. includes

+ command
+ description of command in text
+ what each parameter is, which parameters are optional/required
+ returns
+ raises
+ example code

see drop function as an example of [structure](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.drop.html)

**Comment by quinnwai** [comment]
Yes 100% agree. Did some work to include more of the inputs and outputs in the Quickstart Guide in #32 but we will need another ticket to enumerate this with all of the flags not just the required params and expected output.
